replica/replica.go

```diff
@@ -238,12 +238,11 @@ func construct(readonly bool, size, sectorSize int64, dir, head string, backingF
 
 	r.insertBackingFile()
 	r.ReplicaType = replicaType
-	logrus.Info("Start reading extents")
-	inject.AddPreloadTimeout()
-	if err := PreloadLunMap(&r.volume); err != nil {
-		return r, fmt.Errorf("failed to load Lun map, error: %v", err)
+	if types.PreloadDuringOpen {
```

Comment:
lets donot use global variables, it introduces potential bugs hard to catch.

Reply:
This variable will be removed in the near future when we remove preload() from open() function.
Multiple function signatures will have to be changed to embed this variable in replica structure. But if you think it might create bugs in short term, I can make this change ?

Reply:
better to pass as argument or in a structure